transit-extensions.proto

```diff
@@ -67,10 +67,30 @@ extend transit_realtime.TripDescriptor {
 }
 
 message TransitStopTimeUpdateExtension {
+    enum PredictionAlgorithm {
+        // Data is not predicted
+        NO_PREDICTION = 0;
```

Comment:
How is that useful? if there's no prediction there should be no stop time update?

Reply:
True, maybe we can change the name to `NOT_SPECIFIED`?
The 0 is there because in prediction engine there is a 0 default value.
In the case of propagateDelay, this is the value that it will take. 
We can also name it `PROPAGATE_DELAY` but I feel that a default of NOT_SPECIFIED covers more potential cases.

Reply:
You can potentially have no default and remove that value entirely

Reply:
Right, I just updated the PR.